policies/modules/ROOT/pages/policies-included-tls-outbound.adoc

```diff
@@ -17,17 +17,30 @@ endif::[]
 |
 |===
 
+[NOTE]
+====
+The TLS policy documentation is only applicable to Local Mode Flex Gateways. To apply TLS in Connected Mode, see xref:gateway::flex-conn-tls-config.adoc[Configuring TLS Context for Flex Gateway in Connected Mode].
+====
+
 == Summary
+Flex Gateway supports outbound Transport Layer Security (TLS) and outbound mutual authentication TLS (mTLS) in Local Mode.
+
+To apply outbound TLS in Local Mode, use the xref:gateway::flex-local-tls-config.adoc[Configuring TLS Context for Flex Gateway in Local Mode] tutorial and refer to the following configuration files in this documentation.
```

Comment:
Not sure but should there be configuration file references after this paragraph since it says "refer to the following..."? This wording is used in some additional places below as well.

Reply:
Yup, there will configuration files here, they're just in partials.